### PR TITLE
Dependency of chef-ruby-lvm-attrib

### PR DESCRIPTION
**Pull request by aivaraslaimikis**
## Pull Request Checklist

Solves issue https://github.com/sensu-plugins/sensu-plugins-lvm/issues/5

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [x] RuboCop passes

- [ ] Existing tests pass 

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+## 0.0.4 - 2017-07-10
```
Comment (majormoses) [inline CHANGELOG.md]: remove version and date, we will add this upon release.

File: bin/metrics-vg-usage.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #! /usr/bin/env ruby
 #  encoding: UTF-8
+
```
Comment (majormoses) [inline bin/metrics-vg-usage.rb]: was this added by accident?

Comment (aivaraslaimikis) [inline bin/metrics-vg-usage.rb]: Rubocop "Add an empty line after magic comments":
http://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Layout/EmptyLineAfterMagicComment
Some repos adding empty line too. Do I need to ignore it?

Comment (majormoses) [inline bin/metrics-vg-usage.rb]: no, just checking.

**Comment by aivaraslaimikis** [comment]
Done, please verify.

**Comment by majormoses** [comment]
I am seeing a weird travis stuff, will figure out deploy issue some time today.

**Comment by majormoses** [comment]
manually pushed release: https://rubygems.org/gems/sensu-plugins-lvm/versions/0.0.4

I see that no one has done the proper setup for this gem to deploy to travis and I will set that up but wanted to unblock you with the version you released.

**Pull request merged**